lib/views/widgets/navigation/bottom_navbar.dart

```diff
@@ -23,36 +23,47 @@ class BottomNavBar extends HookConsumerWidget {
       floatingActionButton: const CenterActionButton(),
       floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
       bottomNavigationBuilder: (_, tabsRouter) {
-        return BottomAppBar(
-          shape: const CircularNotchedRectangle(),
-          notchMargin: 5,
-          child: Row(
-            mainAxisSize: MainAxisSize.max,
-            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-            children: <Widget>[
-              NavigationButton(
-                icon: Icons.home_outlined,
-                activeIcon: Icons.home,
-                route: Routes.dashboard,
-              ),
-              NavigationButton(
-                icon: Icons.signal_cellular_0_bar_outlined,
-                activeIcon: Icons.signal_cellular_4_bar,
-                route: Routes.transactions,
-              ),
-              const SizedBox(width: 75),
-              NavigationButton(
-                icon: Icons.notifications_none_outlined,
-                activeIcon: Icons.notifications_active,
-                route: Routes.notifications,
-              ),
-              NavigationButton(
-                icon: Icons.person_outline_outlined,
-                activeIcon: Icons.person,
-                route: Routes.settings,
+        return Container(
+          height: 80,
+          decoration: const BoxDecoration(
+            boxShadow: [
+              BoxShadow(
+                blurRadius: 100.0,
+                offset: Offset(0, 20.0),
               ),
             ],
           ),
+          child: BottomAppBar(
+            shape: const CircularNotchedRectangle(),
+            notchMargin: 5,
```

Comment:
suggesting to change notchMargin to 10 since it looks to close to the navbar compared to the design

(current)
![image](https://user-images.githubusercontent.com/86587091/164420299-5f0431f9-4e89-4a02-b2d6-a9293164c8ef.png)
 
(design)
![image](https://user-images.githubusercontent.com/86587091/164420437-e3fda0fe-cdef-45fc-ac93-5c88c927e330.png)